app/controllers/tasks.py

```diff
@@ -0,0 +1,92 @@
+from flask import abort, current_app
+from flask_restful import marshal_with, reqparse
+from sqlalchemy.exc import DataError, IntegrityError
+import re
+
+from app.controllers.controller import Base
+from app.controllers.marshalling_fields import *
+from app.models.user import Task, db, UserAndTaskRelation
+
+
+post_task = reqparse.RequestParser()
```

Comment:
please be consistent with users controller.

1. parsers and marshals should have use the same naming conventions.
2. parsers and marshals should be defined or in controllers or in extra modules you've created
3. DRY - condition `is user assigned to task` should be moved to function or method and used in all needed places.
4. condition `is user assigned to task` should be rewritten. Image our system has 1KK users, so to verify access you will select all of them from db and loop through them. I think it is not optimal solution.